### PR TITLE
Move style guide entries to be in alphabetical order

### PR DESCRIPTION
**Pull request by andy-armstrong**
This fixes the alphabetical order of the words in the style guide.
### Reviewers
- [x] @edx/doc

FYI: Tag anyone else who might be interested in this PR here.
### Post-review
- [ ] Squash commits

### CODE REVIEW

File: en_us/edx_style_guide/source/wordlist.rst [hunk 1]
```diff
@@ -31,6 +31,8 @@ Word List
        or when using title capitalization. Use email to modify a noun. For
        example, prefer "Send an email message" to "Send an email". Do not use
        as a verb.
+   * - GitHub
+     - The "H" is always capitalized.
    * - ID
      - Capitalize. Use instead of "id" or "Id".
    * - instructor dashboard
```
Comment (lamagnifica) [inline en_us/edx_style_guide/source/wordlist.rst:43]: I don't suppose you'd be willing to alphabetize others VV too, would you??? edX mobile app should go before email

**Review by lamagnifica — APPROVED** [review]
great work! if you have the bandwidth to correct another erroneously positioned entry, that would be great too...!

File: en_us/edx_style_guide/source/wordlist.rst (hunk 1 above)
Comment (andy-armstrong) [reply en_us/edx_style_guide/source/wordlist.rst:43]: Good catch! Done.

**Comment by lamagnifica** [comment]
👍 merge at your leisure!

**Pull request merged**